hc/api/views.py

```diff
@@ -291,6 +295,61 @@ def pings(request, code):
 
     return JsonResponse({"pings": dicts})
 
+@cors("GET")
+@csrf_exempt
+@validate_json()
+@authorize_read
+def flips(request, code):
+    check = get_object_or_404(Check, code=code)
+    if check.project_id != request.project.id:
+        return HttpResponseForbidden()
+
+    if all(x not in request.GET for x in ('start','end','seconds')):
+        flips = Flip.objects.select_related("owner").filter(
+            owner=check, new_status__in=("down","up"),
+        ).order_by("created")
+    else:
+        if any(x in request.GET for x in ('start','end')) and 'seconds' in request.GET:
```

Comment:
Can we rewrite this (and the previous `if`) in a simpler and a dumber way?
IIUC this says "if (either start or end is present) *and* seconds is present that's an error". A simpler way to express that would be:

    if "seconds" in request.GET and ("start" in request.GET or "end" in request.GET):